app/views/events/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+  <div class="col-md-6 form-new-user col-md-offset-3">
+    
+    <%= form_for(@event) do |f| %>
+      <%= f.label :title %><br>
+      <%= f.text_area :title, class: "form-control" %><br>
+      <%= f.label :description %><br>
+      <%= f.text_area :description, class: "form-control" %><br>
+      <%= f.label :location %><br>
+      <%= f.text_area :location, class: "form-control" %><br>
+      <%= f.label :date %><br>
+      <%= f.text_area :date, class: "form-control" %><br>
```

Comment:
but why is date a text_area? try `f.date_field`

Reply:
Fixed